cuqi/pde/_pde.py

```diff
@@ -64,6 +66,13 @@ def _compare_grid(grid1,grid2):
 
         return equal_arrays
 
+    @property
+    def _non_default_args(self):
```

Comment:
It seems PDE, RV, Model and Distribution are all using this pattern now :) We probably have an underlying datastructure hiding in these classes. Thoughts?